### ISSUE TITLE
Android 14 it is not working

### ISSUE THREAD

**Issue by Deepak9811** [mdrajibsk8/react-native-device-number#19]
Not working below code on Android 14

private fun getPhone() {
        val googleApiClient = GoogleApiClient.Builder(this)
                .addApi(Auth.CREDENTIALS_API)
                .addConnectionCallbacks(this)
                .addOnConnectionFailedListener(this)
                .build()
        googleApiClient.connect()
        val hintRequest = HintRequest.Builder()
                .setPhoneNumberIdentifierSupported(true)
                .build()
        val intent = Auth.CredentialsApi.getHintPickerIntent(googleApiClient, hintRequest)
        try {
            startIntentSenderForResult(intent.intentSender, PERMISSION_REQUEST_CODE, null, 0, 0, 0)
        } catch (e: IntentSender.SendIntentException) {
            e.printStackTrace()
        }
    }

**Comment by sanahrGMH** [comment]
Hi @Deepak9811 ,
Had you found any solution for that.

**Comment by Deepak9811** [comment]
> Hi [@Deepak9811](https://github.com/Deepak9811) , Had you found any solution for that.

No, but I'm using the react-native-sim-cards-manager and making a custom alert(modal/popup)
